pkg/volume/csi/csi_mounter.go

```diff
@@ -197,7 +197,7 @@ func (c *csiMountMgr) SetUpAt(dir string, fsGroup *int64) error {
 		accessMode,
 		c.volumeInfo,
 		attribs,
-		"ext4", //TODO needs to be sourced from PV or somewhere else
+		csiSource.FSType,
```

Comment:
Comment above says that `Implicitly inferred to be "ext4" if unspecified.` We need some code for that if `csiSource.FSType` is empty.

Reply:
I saw that in `drivers` repo, we would defaults `FSType` to `ext4`  in function `formatAndMount` if it was not set.  Should we deal with this here ?
btw, when this gets merged, i will update the external provisioner to add `FSType` for newly created PV if it is set in StorageClass parameters.

Reply:
We don't have control about all CSI drivers in the wold to fall back to ext4 when nothing is specified.  Other drivers may choose a different default, so it's Kubernetes who should enforce ext4 here.

Reply:
done, thanks, PTAL